magprime/model_checks.py

```diff
@@ -3,13 +3,13 @@
 
 @prereg_validation.Attendee
 def child_badge_over_13(attendee):
-    if attendee.badge_type == c.CHILD_BADGE and attendee.age_group_conf['val'] not in [c.UNDER_6, c.UNDER_13]:
+    if attendee.is_new and attendee.badge_type == c.CHILD_BADGE and attendee.age_group_conf['val'] not in [c.UNDER_6, c.UNDER_13]:
```

Comment:
This (and the change to the validation message below) ensures that, for example, an attendee with a comped badge can change their DOB.